internal/lib/sandbox/namespaces_linux.go

```diff
@@ -176,7 +176,7 @@ func (n *Namespace) Remove() error {
 	// /var/run/$NSTYPEns/$NSTYPE-d08effa-06eb-a963-f51a-e2b0eceffc5d
 	// but /var/run on most system is symlinked to /run so we first resolve
 	// the symlink and then try and see if it's mounted
-	fp, err := symlink.FollowSymlinkInScope(n.Path(), "/")
+	fp, err := securejoin.SecureJoin("/", n.Path())
```

Comment:
@saschagrunert I am guessing, without any real evidence, that the issue is here. Can you try keeping the symlink package and this function to see? 

I think it's happening on jenkins and not circle because jenkins is forced to have manage_ns_lifecycle=true

This also could be a new interaction between cri-tool master and manage_ns_lifecycle true that doesn't have to do with this pr. but reverting this change will help narrow that down

Reply:
Yeah I was wondering why because the resulting namespace paths are correct. I added another commit on top to revert that change. Let's see what the tests say. :+1: